06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,193 @@
+/**
```

Comment:
I have looked through the file and corrections have been made to necessary parts.